dashboard/src/components/dashboard/GuildList.vue

```diff
@@ -19,7 +19,7 @@
 	<div>
 		<div class="is-size-1">Please select the guild you want to edit.</div>
 		<div id="guildlist" class="columns is-centered is-multiline" :class="{ 'is-loading': !guildsLoaded }">
-			<div class="guildlist column is-half" v-for="guild in guilds" :key="guild.discordId">
+			<div class="guildlist column is-one-fifth" v-for="guild in guilds" :key="guild.discordId">
```

Comment:
This is something I touched as well and burned my fingers in the past. The problem is when the width is just larger than mobile: 
![image](https://user-images.githubusercontent.com/6048348/86515800-f0b4da00-be1b-11ea-898e-9a1becf51942.png)

I think it needs more of these: https://bulma.io/documentation/columns/responsiveness/#different-column-sizes-per-breakpoint

Reply:
This could probably be fixed with screensize specific definitions, but again, this is related to responsive view, which is a drag to do and should be addressed in a separate issue in my opinion - due to the clutter it creates.

Reply:
```suggestion
			<div class="guildlist column is-one-third-tablet is-one-fifth-desktop" v-for="guild in guilds" :key="guild.discordId">
```